_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Introduction to Mixing Audio - How to not sound like a Dial Up Modem"
+author: flavius
+categories: [ audio ]
+image: "mixer.webp"
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
+How much does it cost?
+
+### - Introduction
+
+Audio mixing, what is it? Audio Mixing is where you take multiple input sources of audio and put them into a single or fewer number output signals. An example of input sources would be a microphone, game audio, stream alert audio. An example of an output signal would be our headphones, speakers, or stream. Can you achieve audio mixing for streams and videos without a digital or physical audio mixer? Mostly, but in practice it is significantly more complicated to route audio properly without using some form of audio mixing software or hardware when dealing with multiple sources.
```

Comment:
Phrasing is a little dense for someone who may not be a native English speaker on this sentence:

"An example of an output signal would be our headphones, speakers, or stream." 

May be better to slightly rephrase it as...

"An example of an output signal would be an audio signal that is sent to our headphones, speakers, or stream."

Reply:
Audio mixing, what is it? Audio Mixing is where you take multiple input sources of audio and put them into one or two different output signals. An example of input sources would be a microphone, game audio, stream alert audio. An example of an output signal would be the signal sent to our headphones, speakers, or stream. Can you achieve audio mixing for streams and videos without a digital or physical audio mixer? Mostly, but in practice it is significantly more complicated to route audio properly without using some form of audio mixing software or hardware when dealing with multiple sources.